### ISSUE TITLE
Batched JSON-RPC calls failing if number too large

### ISSUE THREAD

**Issue by markallisongit** [bitcoin-sv/bitcoin-sv#69]
#### If so, please list the steps to reproduce below:
1. Unzip powershell script on Windows
2. Run the script in the repro below by supplying the following params: URI to bitcoin node, RPC username and password
3. e.g. 
```
.\ReproBug66.ps1 -uri "10.0.1.25" -rpcUserName "rpcuser" -rpcPassword "
rpcpassword" -outputFileFullPath "C:\Temp\blockhashes.txt"
```

### Expected behaviour
A text file of block hashes from block 90000 to 92000 should be output.

### Actual behaviour
500 Server error 
```
Invoke-RestMethod : {"result":null,"error":{"code":-32700,"message":"Parse error"},"id":null}

At line:28 char:5
     Invoke-RestMethod -Uri $global:bitcoinRpcUri -Method Post -Conten ...
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

### What version of bitcoin-sv are you using?
0.2.0

### Machine specs:
- OS: CentOS 7
- CPU: 4 vCPU
- RAM: 32 GB
- Disk size: 1 TB
- Disk Type (HD/SDD): HDD

Here's a PowerShell repro. https://www.bitcoinfiles.org/ea734f3de206ca3f85e49acb2686bf744bb987a8470ed55487fee8652743c01d

**Comment by markallisongit** [comment]
Same script works fine on 0.1.1

**Comment by Emobe** [comment]
this is the vision of satoshi so it's working as intended

**Comment by Danconnolly** [comment]
Thanks, we'll check.

**Comment by markallisongit** [comment]
Hi, which version will this be fixed in?

**Comment by Danconnolly** [comment]
This is fixed in the next version, v0.2.1. This is due to be released within two weeks.
